compiler/rustc_feature/src/active.rs

```diff
@@ -161,6 +161,9 @@ declare_features! (
     (active, profiler_runtime, "1.18.0", None, None),
     /// Allows using `rustc_*` attributes (RFC 572).
     (active, rustc_attrs, "1.0.0", None, None),
+    /// Allows using internal rustdoc features like `doc(primitive)` or `doc(keyword)`
+    /// without a future-incompat warning.
```

Comment:
Only `doc(primitive)` is a future-incompat warning. To avoid this getting out of date, let's just do this:

```suggestion
    /// Allows using internal rustdoc features like `doc(primitive)` or `doc(keyword)`.
```